src/videotile/DefaultVideoTile.ts

```diff
@@ -13,6 +13,8 @@ import VideoTileState from './VideoTileState';
 export default class DefaultVideoTile implements DevicePixelRatioObserver, VideoTile {
   private tileState: VideoTileState = new VideoTileState();
 
+  static keepLastFrameWhenPause: boolean = false;
```

Comment:
Hopefully, using a `static` property is a temporary solution until the next major version. One `static` property change applies to all different meeting sessions on the same browser tab, while the JS SDK generally offers meeting-specific settings.

Reply:
If its a temporary solution until the next major version (where i assume at the least, this would just be the default behavior), why don't we just stick it in `MeetingSessionConfiguration`?  Doesn't seem like we need to be this clever unless we are doing it intentionally.  We could even mark it as 'Recommended' in the doc string (which I wish existed here explaining why to use it or not).

Reply:
I actually started out as a meeting session config but the default video tile does not have access to the config object. One way is try to get it from video tile controller but then it is not available in these static methods `disconnectVideoStreamFromVideoElement`. I am open to suggestion.

This is probably fine since I think this setting varies from app to app and not meeting to meeting.

Reply:
This may sound silly, but could we just read it from meeting session configuration and then set this static variable underneath the hood?  Its a total hack (which will eventually be removed), but it allows us to better document and expose the option which is likely to be a popular one.